tests/debugging/exploration/test_bootstrap.py

```diff
@@ -60,3 +60,33 @@ def test_exploration_file_output():
     from tests.debugging.exploration._config import config
 
     assert config.output_file == Path("expl.txt")
+
+
+@pytest.mark.subprocess(
+    parametrize=dict(
+        DD_REMOTE_CONFIGURATION_ENABLED=["1", "0"],
+    ),
+)
+def test_rc_default_products_registered():
```

Comment:
I don't think it makes sense for this test to live in the `debugging` directory, because that's about the Debugger product. The test might be more at home in https://github.com/DataDog/dd-trace-py/tree/main/tests/internal/remoteconfig

Reply:
Makes sense, I'll move the test in there